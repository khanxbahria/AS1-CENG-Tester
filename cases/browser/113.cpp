#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("v", "xx","www");
Tab B = Tab("iii", "s","z");
Tab C = Tab("ee", "r","pp");
Tab D = Tab("hhh", "sss","b");
Tab E = Tab("x", "qq","b");
Tab F = Tab("xxx", "mmm","zz");
Tab G = Tab("hh", "aaa","uu");
Tab H = Tab("ii", "m","y");
Tab I = Tab("ccc", "l","e");
Tab J = Tab("iii", "c","ee");
Tab K = Tab("j", "oo","uuu");
Tab L = Tab("q", "mmm","c");
Tab M = Tab("n", "fff","d");
Tab N = Tab("ff", "ddd","ww");
Tab O = Tab("bb", "f","pp");
Tab P = Tab("ll", "jj","hhh");
Tab Q = Tab("m", "bbb","uuu");
Tab R = Tab("ee", "uu","ii");
Tab S = Tab("zz", "kk","c");
Tab T = Tab("uuu", "e","ii");
Tab U = Tab("ww", "ss","ccc");
Tab V = Tab("d", "c","j");
Tab W = Tab("i", "b","ttt");
Tab X = Tab("i", "jjj","yyy");
Tab Y = Tab("ccc", "q","j");
Tab Z = Tab("nnn", "pp","w");
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.newTab(A);
b.changeActiveTabTo(3);
c.newTab(B);
d.changeActiveTabTo(3);
a.newTab(C);
b.newTab(D);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.newTab(E);
b.newTab(F);
c.changeActiveTabTo(5);
d.newTab(G);
a.changeActiveTabTo(0);
b.newTab(H);
c.newTab(I);
d.newTab(J);
a.changeActiveTabTo(4);
b.newTab(K);
c.changeActiveTabTo(4);
d.newTab(L);
a.changeActiveTabTo(2);
b.newTab(M);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.newTab(N);
b.newTab(O);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.newTab(P);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(4);
b.newTab(Q);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.newTab(R);
b.changeActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(3);
b.newTab(S);
c.newTab(T);
d.changeActiveTabTo(3);
a.changeActiveTabTo(4);
b.newTab(U);
c.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.newTab(V);
d.moveActiveTabTo(2);
a.moveActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.newTab(W);
b.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.newTab(X);
a.changeActiveTabTo(3);
b.newTab(Y);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.newTab(Z);
d.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(1);
b.moveActiveTabTo(1);
c.changeActiveTabTo(1);
b.changeActiveTabTo(2);
a.changeActiveTabTo(2);
c.changeActiveTabTo(2);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
d.changeActiveTabTo(0);
a.changeActiveTabTo(5);
d.moveActiveTabTo(5);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.changeActiveTabTo(4);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(3);
c.moveActiveTabTo(3);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.changeActiveTabTo(5);
a.moveActiveTabTo(0);
b.changeActiveTabTo(0);
c.moveActiveTabTo(0);
a.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.moveActiveTabTo(4);
c.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.moveActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.changeActiveTabTo(0);
a.moveActiveTabTo(1);
b.changeActiveTabTo(1);
c.moveActiveTabTo(1);
a.changeActiveTabTo(0);
b.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(4);
c.moveActiveTabTo(4);
a.moveActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
b.moveActiveTabTo(2);
c.moveActiveTabTo(2);
a.moveActiveTabTo(4);
a.changeActiveTabTo(3);
b.moveActiveTabTo(3);
d.changeActiveTabTo(3);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(1);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
a.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(3);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->getWindow(1);
brows->getWindow(2);
brows->switchToWindow(1);
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->switchToWindow(4);
brows->getWindow(9);
brows->switchToWindow(4);
brows->switchToWindow(1);
brows->closeEmptyWindows();
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->print();
brows->closeWindow();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(6);
brows->print();
brows->newWindow();
brows->newWindow();
brows->newWindow();
brows->print();
brows->print();
brows->mergeAllWindows();
brows->closeWindow();
brows->print();
brows->switchToWindow(0);
brows->switchToWindow(2);
brows->closeWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(2);
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(7);
brows->getWindow(7);
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(4);
brows->print();
brows->switchToWindow(3);
brows->mergeAllWindows();
brows->print();
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->getWindow(5);
brows->closeEmptyWindows();
brows->getWindow(1);
brows->getWindow(5);
brows->switchToWindow(4);
brows->closeEmptyWindows();
brows->closeWindow();
brows->newWindow();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->closeEmptyWindows();
brows->getWindow(0);
brows->print();
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(8);
brows->print();
brows->mergeAllWindows();
brows->switchToWindow(6);
brows->newWindow();
brows->newWindow();
brows->closeWindow();
brows->newWindow();
brows->closeWindow();
brows->getWindow(1);
brows->closeEmptyWindows();
brows->newWindow();
brows->getWindow(1);
brows->print();
brows->print();
brows->closeWindow();
brows->closeWindow();
brows->switchToWindow(2);
brows->newWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->getWindow(4);
brows->switchToWindow(4);
brows->getWindow(5);
brows->newWindow();
brows->print();
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(2);
brows->print();
brows->mergeAllWindows();
brows->mergeAllWindows();
brows->newWindow();
brows->newWindow();
brows->getWindow(4);
brows->switchToWindow(6);
brows->getWindow(3);
brows->print();
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->print();
brows->switchToWindow(4);
brows->closeWindow();
brows->getWindow(9);
brows->closeWindow();
brows->getWindow(1);
brows->closeEmptyWindows();
brows->getWindow(0);
brows->closeWindow();
delete brows;
}
	return 0;
}