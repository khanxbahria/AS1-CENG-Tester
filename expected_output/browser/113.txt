The active tab is tab 3
r
ee
pp

xx
v
www

ddd
ff
ww





b
i
ttt

jj
ll
hhh

uu
ee
ii

qq
x
b

The active tab is tab 0




The active tab is tab 0
mmm
xxx
zz

kk
zz
c





q
ccc
j

ss
ww
ccc

sss
hhh
b

f
bb
pp

bbb
m
uuu

m
ii
y

oo
j
uuu

fff
n
d

The active tab is tab 4
l
ccc
e

pp
nnn
w

e
uuu
ii

s
iii
z





c
d
j

The active tab is tab 4




aaa
hh
uu

jjj
i
yyy

mmm
q
c

c
iii
ee

The active tab is tab 31




r
ee
pp

xx
v
www

ddd
ff
ww









mmm
xxx
zz

kk
zz
c





q
ccc
j

ss
ww
ccc

sss
hhh
b

f
bb
pp

bbb
m
uuu

m
ii
y

oo
j
uuu

fff
n
d

l
ccc
e

pp
nnn
w

e
uuu
ii

s
iii
z





c
d
j





aaa
hh
uu

jjj
i
yyy

mmm
q
c

c
iii
ee

b
i
ttt

jj
ll
hhh

uu
ee
ii

qq
x
b

The browser is empty
The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 0




The active tab is tab 0




Window is empty
The list is empty.
The active tab is tab 1








Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 1








Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 1








Window is empty
The list is empty.
Window is empty
The list is empty.
